project/Dependencies.scala

```diff
@@ -2,6 +2,14 @@ import sbt._
 
 object Dependencies {
 
+  val akkaUtil: Seq[ModuleID] = {
+    val akkaVersion = "2.6.9"
```

Comment:
It looks like we two akkaVersion declared. Maybe it would make sense to have 1

Reply:
Sure, done.